src/util/azure-auth.js

```diff
@@ -0,0 +1,27 @@
+const { WorkloadIdentityCredential } = require("@azure/identity");
+
+let azureAuth
+
+const makeAzureAuth = async () => {
+    const scope = 'https://ossrdbms-aad.database.windows.net/.default'
+
+    // This relies on environment variables that get injected.
+    // AZURE_AUTHORITY_HOST:       (Injected by the webhook)
+    // AZURE_CLIENT_ID:            (Injected by the webhook)
+    // AZURE_TENANT_ID:            (Injected by the webhook)
+    // AZURE_FEDERATED_TOKEN_FILE: (Injected by the webhook)
+
+    const credential = new WorkloadIdentityCredential()
+    const tokenResponse = await credential.getToken(scope)
+    const accessToken = tokenResponse.token
+    return {
+        dbPassword: accessToken
```

Comment:
Dit werkt helaas toch niet. Mijn fout. De accesstoken moet hier opgehaald worden anders verloopt hij op een gegeven moment.
Ben er ook mee bezig, denk dat dit al een stap verder is maar weet nog niet of het werkt: https://github.com/Amsterdam/digitaleinkoopassistent-backend/pull/8/files

Reply:
Aangepast!